src/wrap-store/wrapStore.js

```diff
@@ -19,8 +19,9 @@ const promiseResponder = (dispatchResult, send) => {
       });
     })
     .catch((err) => {
+      console.error(err);
       send({
-        error: err,
+        error: err.message,
```

Comment:
What is the reason to only pass the error message in this case? I personally extend my errors to have other properties in my own work.